[SNIPER] (0) Using trace file /home_masters/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
[SNIPER] (1) Using trace file /home_masters/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
[SNIPER] (2) Using trace file /home_masters/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
[SNIPER] (3) Using trace file /home_masters/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
[SNIPER] Start
----------BEGIN_INIT_STATS_MANAGER------------
----------END_INIT_STATS_MANAGER------------
[DvfsManager] Core 0 at 2.66 GHz (global clock 2.66 GHz)
[DvfsManager] Core 1 at 2.66 GHz (global clock 2.66 GHz)
[DvfsManager] Core 2 at 2.66 GHz (global clock 2.66 GHz)
[DvfsManager] Core 3 at 2.66 GHz (global clock 2.66 GHz)
Normal Cache Total set: 524288
[Ramulator config] SpeedEntry: nRFC(130), nREFI(3900), nREFI1B(256)
Each level size: 
32 4 2 2048 8192 
Here we initilize a off-chip model!
[Ramulator config] SpeedEntry: nRFC(130), nREFI(3900), nREFI1B(256)
Each level size: 
1 1 32 16384 8192 
now initialize a stacked dram unison cache
Executing Python script /scratch/miz087/results/isca/10-20-test_cache_cpu_remap0_t104_spec/sim.scripts.py
[SNIPER] --------------------------------------------------------------------------------
[SNIPER] Sniper using SIFT/trace-driven frontend
[SNIPER] Running full application in DETAILED mode
[SNIPER] --------------------------------------------------------------------------------

************recordStats once at 0
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 0, Dynamic: 0
[Warning] the power of processor is too small, we skip the temperature calculation!
[SNIPER] Enabling performance models

************recordStats once at 0
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 0, Dynamic: 0
[Warning] the power of processor is too small, we skip the temperature calculation!
[SNIPER] Setting instrumentation mode to DETAILED
[TRACE:0] [HOOKS] Entering ROI
[TRACE:2] [HOOKS] Entering ROI
[TRACE:3] [HOOKS] Entering ROI
[TRACE:1] [HOOKS] Entering ROI
[TRACE:0] Delta   =     1.000000e-06
[TRACE:0] Epsilon =     1.000000e-17
[TRACE:0] Leaving algorithm
[TRACE:0] Column representation
[TRACE:0] Forest-Tomlin update
[TRACE:0] Steep pricing
[TRACE:0] Fast ratiotest
[TRACE:0] No scaling
[TRACE:0] General simplifier
[TRACE:0] Weight starter
[TRACE:0] loading LP file pds-50.mps
[TRACE:2] Delta   =     1.000000e-06
[TRACE:3] Delta   =     1.000000e-06
[TRACE:1] Delta   =     1.000000e-06
[TRACE:2] Epsilon =     1.000000e-17
[TRACE:3] Epsilon =     1.000000e-17
[TRACE:2] Leaving algorithm
[TRACE:3] Leaving algorithm
[TRACE:2] Column representation
[TRACE:3] Column representation
[TRACE:1] Epsilon =     1.000000e-17
[TRACE:2] Forest-Tomlin update
[TRACE:3] Forest-Tomlin update
[TRACE:1] Leaving algorithm
[TRACE:1] Column representation
[TRACE:1] Forest-Tomlin update
[TRACE:2] Steep pricing
[TRACE:3] Steep pricing
[TRACE:2] Fast ratiotest
[TRACE:3] Fast ratiotest
[TRACE:2] No scaling
[TRACE:1] Steep pricing
[TRACE:3] No scaling
[TRACE:1] Fast ratiotest
[TRACE:2] General simplifier
[TRACE:3] General simplifier
[TRACE:1] No scaling
[TRACE:2] Weight starter
[TRACE:3] Weight starter
[TRACE:1] General simplifier
[TRACE:2] loading LP file pds-50.mps
[TRACE:3] loading LP file pds-50.mps
[TRACE:1] Weight starter
[TRACE:1] loading LP file pds-50.mps

************recordStats once at 999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 0, Dynamic: 0
[Warning] the power of processor is too small, we skip the temperature calculation!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 179, in parse_results_from_dir
    results += stats.parse_stats((k1, k2), ncores, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 8, in parse_stats
    v2 = self.read_snapshot(k2, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 44, in read_snapshot
    raise ValueError('Invalid prefix %s' % prefix)
ValueError: Invalid prefix energystats-tempA

************recordStats once at 1999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 0, Dynamic: 0
[Warning] the power of processor is too small, we skip the temperature calculation!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 179, in parse_results_from_dir
    results += stats.parse_stats((k1, k2), ncores, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 8, in parse_stats
    v2 = self.read_snapshot(k2, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 44, in read_snapshot
    raise ValueError('Invalid prefix %s' % prefix)
ValueError: Invalid prefix energystats-tempB

************recordStats once at 3000
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
*[Hotspot] begin initialization!
[DTM Trigger] CPU_MAX(THRES): 72.960(104.000), DRAM_MAX(THRES): 62.683(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 179, in parse_results_from_dir
    results += stats.parse_stats((k1, k2), ncores, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 8, in parse_stats
    v2 = self.read_snapshot(k2, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 44, in read_snapshot
    raise ValueError('Invalid prefix %s' % prefix)
ValueError: Invalid prefix energystats-tempA
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 4000
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 77.806(104.000), DRAM_MAX(THRES): 65.733(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 179, in parse_results_from_dir
    results += stats.parse_stats((k1, k2), ncores, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 8, in parse_stats
    v2 = self.read_snapshot(k2, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 44, in read_snapshot
    raise ValueError('Invalid prefix %s' % prefix)
ValueError: Invalid prefix energystats-tempA
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 4999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 80.867(104.000), DRAM_MAX(THRES): 68.166(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 5999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 83.275(104.000), DRAM_MAX(THRES): 70.133(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 6997
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 85.334(104.000), DRAM_MAX(THRES): 71.814(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 7999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 87.156(104.000), DRAM_MAX(THRES): 73.298(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 8999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 88.794(104.000), DRAM_MAX(THRES): 74.630(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 9972
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 90.278(104.000), DRAM_MAX(THRES): 75.836(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 10987
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 91.628(104.000), DRAM_MAX(THRES): 76.934(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 11995
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 92.859(104.000), DRAM_MAX(THRES): 77.936(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 12959
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 93.983(104.000), DRAM_MAX(THRES): 78.851(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 13981
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 95.012(104.000), DRAM_MAX(THRES): 79.688(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 14921
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 95.953(104.000), DRAM_MAX(THRES): 80.456(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 15999
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 96.816(104.000), DRAM_MAX(THRES): 81.159(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 16963
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 97.606(104.000), DRAM_MAX(THRES): 81.804(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 17896
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 98.331(104.000), DRAM_MAX(THRES): 82.396(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 18892
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
[DTM Trigger] CPU_MAX(THRES): 98.996(104.000), DRAM_MAX(THRES): 82.940(85.000)
	Current DTM method: 1
Here we try remapping!
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 178, in parse_results_from_dir
    stats = sniper_stats.SniperStats(resultsdir)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 57, in SniperStats
    stats = sniper_stats_sqlite.SniperStatsSqlite(os.path.join(resultsdir, 'sim.stats.sqlite3'))
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 7, in __init__
    self.names = self.read_metricnames()
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 20, in read_metricnames
    c.execute('select nameid, objectname, metricname from `names`')
sqlite3.OperationalError: database is locked
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 167, in hook_periodic
    self.callback(time, time_delta)
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 87, in periodic
    self.update()
  File "/home_masters/miz087/workplace/micro_exp/scripts/thermalstats.py", line 118, in update
    sim.util.db_delete(self.name_last)
  File "/home_masters/miz087/workplace/micro_exp/scripts/sim/util.py", line 204, in db_delete
    cursor.execute('DELETE FROM prefixes WHERE prefixid = ?', (prefixid[0][0],))
sqlite3.OperationalError: database is locked
Vault num: 32, Bank num: 8, Row num: 2048
Vault#0 Delay: 0
Vault#1 Delay: 0
Vault#2 Delay: 0
Vault#3 Delay: 0
Vault#4 Delay: 0
Vault#5 Delay: 0
Vault#6 Delay: 0
Vault#7 Delay: 0
Vault#8 Delay: 0
Vault#9 Delay: 0
Vault#10 Delay: 0
Vault#11 Delay: 0
Vault#12 Delay: 0
Vault#13 Delay: 0
Vault#14 Delay: 0
Vault#15 Delay: 0
Vault#16 Delay: 0
Vault#17 Delay: 0
Vault#18 Delay: 0
Vault#19 Delay: 0
Vault#20 Delay: 0
Vault#21 Delay: 0
Vault#22 Delay: 0
Vault#23 Delay: 0
Vault#24 Delay: 0
Vault#25 Delay: 0
Vault#26 Delay: 0
Vault#27 Delay: 0
Vault#28 Delay: 0
Vault#29 Delay: 0
Vault#30 Delay: 0
Vault#31 Delay: 0
-----and here we finish handling remapping results in cache controller! the Latency: 0
Bus Delay: 0, Process Latency: 0
---invalid_cnt: 0, total: 524288

************recordStats once at 19920
[Warning] power scale is less than 1, please check!
[Overall Power Consumption] Peak: 26.4939, Dynamic: 12.7852
error: invalid trace file format
Traceback (most recent call last):
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 1321, in <module>
    main(jobid = jobid, resultsdir = resultsdir, powertype = powertype, config = config, outputfile = outputfile, no_graph = no_graph, print_stack = not no_text, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/mcpat.py", line 141, in main
    results = sniper_lib.get_results(jobid, resultsdir, partial = partial)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 47, in get_results
    results = parse_results_from_dir(resultsdir, partial = partial, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_lib.py", line 179, in parse_results_from_dir
    results += stats.parse_stats((k1, k2), ncores, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats.py", line 8, in parse_stats
    v2 = self.read_snapshot(k2, metrics = metrics)
  File "/home_masters/miz087/workplace/micro_exp/tools/sniper_stats_sqlite.py", line 44, in read_snapshot
    raise ValueError('Invalid prefix %s' % prefix)
ValueError: Invalid prefix roi-end
[SNIPER] End
[SNIPER] Elapsed time: 581.46 seconds
[SNIPER] Running McPAT
--no-roi 
--power 
-n 4
-c cache_cpu_remap0
-d /scratch/miz087/results/isca/10-20-test_cache_cpu_remap0_t104_spec
-T /home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
hehe /home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
-g --perf_model/thermal/cpu_temp_thres=104
-g --perf_model/remap_config/on_chip_wb=false
copy HotSpot and ramulator files to /scratch/miz087/results/isca/10-20-test_cache_cpu_remap0_t104_spec
haha: /home_masters/miz087/workplace/micro_exp/run-sniper -n 4 -m 'localhost' -d '/scratch/miz087/results/isca/10-20-test_cache_cpu_remap0_t104_spec' -sthermalstats --power -c cache_cpu_remap0 -g --perf_model/thermal/cpu_temp_thres=104 -g --perf_model/remap_config/on_chip_wb=false --curdir=/home_masters/miz087/workplace/micro_exp/benchmarks --traces=/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift,/home/miz087/workplace/micro_exp/benchmarks/traces/soplex/soplex.0.sift
